### ISSUE TITLE
Feature Request: Support PSDefaultValue, SupportsWildcards in ExtendedCmdletHelpInfo

### ISSUE THREAD

**Issue by juneb** [PowerShell/PowerShell#2490]
The ExtendedCmdletHelpInfo object that Get-Help returns when there's no help for the cmdlet (auto-generated help) doesn't have DefaultValue or Accept Wildcard Characters properties on the parameter property object.

As a result, if you don't write help, but include the PSDefaultValue or SupportsWildcards attributes, they are ignored.

```
function Test-Something
{
    param
    (
        [Parameter(Mandatory = $false)]
        [PSDefaultValue(Help = 'Name of the local computer.')]
        [String]
        $ComputerName = $env:COMPUTERNAME,

        [Parameter(Mandatory = $true)]
        [SupportsWildcards()]
        [string]
        $Path
    )

    "Testing default values on $ComputerName and wildcard support on $Path."
}
Get-Help Test-Something -Parameter *

-ComputerName <string>                         

    Required?                    false         
    Position?                    0             
    Accept pipeline input?       false         
    Parameter set name           (All)         
    Aliases                      None          
    Dynamic?                     false         


-Path <string>                                 

    Required?                    true          
    Position?                    1             
    Accept pipeline input?       false         
    Parameter set name           (All)         
    Aliases                      None          
    Dynamic?                     false         
```

**Comment by msftrncs** [comment]
@SteveL-MSFT, this seems already supported in 6.2.  I just used both in a script and checked the help and they were there:

```PowerShell
Param(
    # Specifies a path to one or more locations to search for ResX files. Wildcards are permitted.
    [Parameter(Position = 0, ValueFromPipeline, ValueFromPipelineByPropertyName)]
    [Alias('Path')]
    [SupportsWildCards()]
    [PSDefaultValue(Help = 'Current directory')]
    [string[]] $SearchPath = '.',

    # Recurse the path(s) to find files.
    [switch] $Recurse,

    # Depth of recursion allowed to find files.
    [uint32] $Depth
)
```
Output from help:
```
-SearchPath <String[]>
    Provides path(s) for which to search for files to reduce.  If the path is a folder, all RESX files in that folder will be reduced.  Only files of type '*.RESX' (not already named '* Reduced.RESX') will be reduced.

    Required?                    false
    Position?                    1
    Default value                Current directory
    Accept pipeline input?       true (ByValue, ByPropertyName)
    Accept wildcard characters?  true


-Recurse [<SwitchParameter>]
    Recurses all specified paths (of which are directories, or result in matching directories) in the search for RESX files to reduce.

    Required?                    false
    Position?                    named
    Default value                False
    Accept pipeline input?       false
    Accept wildcard characters?  false


-Depth <UInt32>
    Used with -Recurse, limits the depth of recursion.

    Required?                    false
    Position?                    named
    Default value                0
    Accept pipeline input?       false
    Accept wildcard characters?  false
```

**Comment by vexx32** [comment]
You'll note that your comments there are indeed considered parameter help values and are picked up by Get-Help. This issue seems to be about the behaviour when PS doesn't notice any comment-help for the function and isn't picking up the additional metadata.

**Comment by msftrncs** [comment]
Ah,  interesting difference.   Even just adding `<# .notes #>` makes all the difference, even though the blank 'notes' does not display.

**Comment by ThomasNieto** [comment]
The `[SupportsWildcards]` issue is fixed with #13353. I can take a look at having `[PSDefaultValue]` adding `defaultValue` and `description` property on the parameter properties.
